torch/_dynamo/variables/builtin.py

```diff
@@ -1323,7 +1323,9 @@ def _grad_changed(old, new):
         elif isinstance(obj, TorchInGraphFunctionVariable):
             # Get OpOverload from an OpOverloadPacket, e.g., torch.ops.aten.add.default.
             member = getattr(obj.value, name)
-            if trace_rules.is_aten_op_or_tensor_method(member):
+            if isinstance(
+                member, torch._ops.OpOverload
```

Comment:
Probably need to include ```OpOverloadPacket```?
https://github.com/pytorch/pytorch/blob/2a63dd888904dab548c565f2e6ef862cfda5287c/torch/_dynamo/utils.py#L528